src/either_or_both.rs

```diff
@@ -29,19 +29,13 @@ impl<A, B> EitherOrBoth<A, B> {
     /// If `Left`, return true. Otherwise, return false.
     /// Exclusive version of [`has_left`](EitherOrBoth::has_left).
     pub fn is_left(&self) -> bool {
-        match *self {
-            Left(_) => true,
-            _ => false,
-        }
+        matches!(*self, Left(_))
```

Comment:
```suggestion
        matches!(self, Left(_))
```

Reply:
Indeed, it compiles to the exact same thing (I just checked a simple example on godbolt).
I'm not sure we should care when I did not find any clippy lint for it.
I'll let the author decide here.

Reply:
yeah, that's definitely a nitpick. on the other hand in a pr which satisfies clippy, it might be kind of future proofing :smile:

Reply:
Sorry thank you done.